src/wallet.rs

```diff
@@ -192,6 +192,10 @@ impl Wallet {
         Ok(enc_sk)
     }
 
+    /// Encrypt the wallet using a password.
+    /// The Utxo cache is dropped.
```

Comment:
Rustdoc convention is to have a newline between the first sentence and the body: https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components